observer/test/platform-stats.test.js

```diff
@@ -37,14 +37,15 @@ describe('platform-stats-generator', () => {
   describe('updateDailyTransferStats', () => {
     it('should correctly update daily Transfer stats with new transfer events', async () => {
       await updateDailyTransferStats(pgPool, { to_address: 'address1', amount: 100 }, 1)
-      await updateDailyTransferStats(pgPool, { to_address: 'address1', amount: 200 }, 1)
+      await updateDailyTransferStats(pgPool, { to_address: 'address1', amount: 200 }, 2)
 
       const { rows } = await pgPool.query(`
-        SELECT day::TEXT, to_address, amount FROM daily_reward_transfers
-        WHERE to_address = $1
-      `, ['address1'])
```

Comment:
No reason to have this WHERE clause, must have been a mistaken addition from before